src/os/bluestore/KernelDevice.cc

```diff
@@ -130,7 +130,6 @@ int KernelDevice::open(const string& p)
   } else {
     size = st.st_size;
   }
-  size &= ~(block_size);
```

Comment:
The block_size can be reconfigured and hence can be any value other than st.st_blksize(see L116).
So we still need to align to block_size here. right?